DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/VerificationPickSupervisor.cshtml

```diff
@@ -63,7 +63,7 @@
     <p>Can't see the  @Model.SelfAssessment.VerificationRoleName you need? <a asp-action="ManageSupervisors" asp-route-selfAssessmentId="@Model.SelfAssessment.Id">Click here</a> to manage supervisors.</p>
     <div class="nhsuk-back-link">
       <a class="nhsuk-back-link__link"
-         asp-action="SelfAssessmentOverview" asp-route-vocabulary="@Model.VocabPlural()"
+         asp-action="ReviewConfirmationRequests"

```

Comment:
We need to use this action on the mobile back link (line 17).
We also need to add an active node to the breadcrumbs "Confirmation requests" with the same action and relabel the final dead node "New request"